Tests/Admin/AdminHelperTest.php

```diff
@@ -25,16 +25,16 @@ class AdminHelperTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+        $container = $this->createMock('Symfony\Component\DependencyInjection\ContainerInterface');
```

Comment:
This won't work for old versions of php. Why don't you use a wrapper, like discussed with @core23 ?